lib/lte_link_control/Kconfig

```diff
@@ -85,6 +85,13 @@ config LTE_PSM_REQ_RAT
 	  See 3GPP 27.007 Ch. 7.38.
 	  And 3GPP 24.008 Ch. 10.5.7.3 for data format.
 
+config LTE_PROPRIETARY_PSM_REQ
+	bool "Enable use of proprietary PSM"
+	help
+	  Enable use of proprietary PSM using AT%FEACONF. To use this feature, also PSM request
+	  must be enabled.
```

Comment:
As `LTE_PSM_REQ` is a requirement, maybe it should be checked either in Kconfig or a build assert that both are enabled

Reply:
I did have a Kconfig dependency in the first place, but decided the remove it. The reason for this decision was that modem allows to enable proprietary PSM also when PSM is not requested. So, one could enable `LTE_PROPRIETARY_PSM_REQ` and then use `lte_lc_psm_req()` to control the PSM is request. When PSM is requested but network does not allow it's usage, modem would use proprietary PSM. If PSM is not requested, modem would obviously not use proprietary PSM either.

Does this sound ok?

Reply:
Yep, that sounds good, thanks for explaining